rollup.config.js

```diff
@@ -31,6 +34,11 @@ export default [
       { file: pkg.main, format: "cjs" },
       { file: pkg.module, format: "es" },
     ],
-    plugins: [typescript({ tsconfig: "./tsconfig.json" })],
+    plugins: [
+        typescript({
+            tsconfig: "./tsconfig.json",
+            exclude: ["**/node_modules/**"]
```

Comment:
Why did you make this change?

Reply:
was for temp work around something, now removed